<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-193VCU3" failures="13" tests="84" name="scripts.HomePage_T_C" time="1252.778" errors="8" timestamp="2024-04-04T11:37:35 IST" skipped="0">
  <testcase classname="scripts.HomePage_T_C" name="verifyProtocol" time="14.949"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyFaviconNetwork" time="10.798"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyOgUrlRepetition" time="30.510"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyOgImageRepetition" time="14.739"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyFaviconNetwork" time="12.586"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyOgImageNetwork" time="9.927"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyHttpInPageSource" time="7.874"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyDescRepetition" time="18.704"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyDomainPipeline" time="11.087"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyOgImageNetwork" time="10.589"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyRobotsBeforeSubmission" time="16.270">
    <failure message="The following asserts failed:
	expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at scripts.HomePage_T_C.verifyRobotsBeforeSubmission(HomePage_T_C.java:20)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1589)
]]>
    </failure>
  </testcase> <!-- verifyRobotsBeforeSubmission -->
  <testcase classname="scripts.HomePage_T_C" name="verifyTitleRepetition" time="20.737"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyRobotsBeforeSubmission" time="7.175">
    <failure message="The following asserts failed:
	expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at scripts.HomePage_T_C.verifyRobotsBeforeSubmission(HomePage_T_C.java:20)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1589)
]]>
    </failure>
  </testcase> <!-- verifyRobotsBeforeSubmission -->
  <testcase classname="scripts.HomePage_T_C" name="verifyDomainPipeline" time="45.411"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyMetaContent" time="9.768">
    <error message="Cannot invoke &quot;java.util.List.get(int)&quot; because the return value of &quot;java.util.Map.get(Object)&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "java.util.List.get(int)" because the return value of "java.util.Map.get(Object)" is null
at utilities.MetaFileHandler.readMetaContentFromExcel(MetaFileHandler.java:99)
at scripts.HomePage_T_C.verifyMetaContent(HomePage_T_C.java:88)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1589)
]]>
    </error>
  </testcase> <!-- verifyMetaContent -->
  <testcase classname="scripts.HomePage_T_C" name="verifyMetaContent" time="17.618">
    <error message="Cannot invoke &quot;java.util.List.get(int)&quot; because the return value of &quot;java.util.Map.get(Object)&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "java.util.List.get(int)" because the return value of "java.util.Map.get(Object)" is null
at utilities.MetaFileHandler.readMetaContentFromExcel(MetaFileHandler.java:99)
at scripts.HomePage_T_C.verifyMetaContent(HomePage_T_C.java:88)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1589)
]]>
    </error>
  </testcase> <!-- verifyMetaContent -->
  <testcase classname="scripts.HomePage_T_C" name="verifyMetaContent" time="20.148">
    <error message="Cannot invoke &quot;java.util.List.get(int)&quot; because the return value of &quot;java.util.Map.get(Object)&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "java.util.List.get(int)" because the return value of "java.util.Map.get(Object)" is null
at utilities.MetaFileHandler.readMetaContentFromExcel(MetaFileHandler.java:99)
at scripts.HomePage_T_C.verifyMetaContent(HomePage_T_C.java:88)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1589)
]]>
    </error>
  </testcase> <!-- verifyMetaContent -->
  <testcase classname="scripts.HomePage_T_C" name="verifyDomainViewPort" time="14.540"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyDescRepetition" time="11.875"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyHttpInPageSource" time="10.889"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyNetworkStatus" time="13.270"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyStyleSheets" time="24.988">
    <failure message="The following asserts failed:
	expected [false] but found [true],
	expected [false] but found [true]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [false] but found [true],
expected [false] but found [true]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at scripts.HomePage_T_C.verifyStyleSheets(HomePage_T_C.java:232)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1589)
]]>
    </failure>
  </testcase> <!-- verifyStyleSheets -->
  <testcase classname="scripts.HomePage_T_C" name="verifyStyleSheets" time="9.826"/>
  <testcase classname="scripts.HomePage_T_C" name="verifySubdomain" time="21.065"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyOgUrlRepetition" time="23.973"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyTitleRepetition" time="12.672"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyNetworkStatus" time="12.009"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyDomainPipeline" time="11.342"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyOgUrlRepetition" time="10.312"/>
  <testcase classname="scripts.HomePage_T_C" name="verifySitemap" time="5.862"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyConsole" time="13.545"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyImageFormat" time="14.550">
    <failure message="The following asserts failed:
	https://worksfyi.com/wf-theme/images/company-logos/rnjz-us_postal_service-logo.png expected [true] but found [false],
	https://worksfyi.com/wf-theme/images/company-logos/rnjz-dominos-logo.png expected [true] but found [false],
	https://worksfyi.com/wf-theme/images/company-logos/rnjz-dollar_tree-logo.png expected [true] but found [false],
	https://worksfyi.com/wf-theme/images/company-logos/rnjz-chewy-logo.png expected [true] but found [false],
	https://worksfyi.com/wf-theme/images/company-logos/rnjz-hilti_north_america-logo.png expected [true] but found [false],
	https://worksfyi.com/wf-theme/images/company-logos/rnjz-petsmart-logo.jpg expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
https://worksfyi.com/wf-theme/images/company-logos/rnjz-us_postal_service-logo.png expected [true] but found [false],
https://worksfyi.com/wf-theme/images/company-logos/rnjz-dominos-logo.png expected [true] but found [false],
https://worksfyi.com/wf-theme/images/company-logos/rnjz-dollar_tree-logo.png expected [true] but found [false],
https://worksfyi.com/wf-theme/images/company-logos/rnjz-chewy-logo.png expected [true] but found [false],
https://worksfyi.com/wf-theme/images/company-logos/rnjz-hilti_north_america-logo.png expected [true] but found [false],
https://worksfyi.com/wf-theme/images/company-logos/rnjz-petsmart-logo.jpg expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at scripts.HomePage_T_C.verifyImageFormat(HomePage_T_C.java:182)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1589)
]]>
    </failure>
  </testcase> <!-- verifyImageFormat -->
  <testcase classname="scripts.HomePage_T_C" name="verifyOgImageNetwork" time="13.343"/>
  <testcase classname="scripts.HomePage_T_C" name="verifySubdomain" time="10.866"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyScrollButton" time="16.281">
    <error message="Expected condition failed: waiting for Attribute or property &#039;class&#039; to be &#039;d-none&#039;. Current value: &#039;btn theme-btn py-2 px-3 position-fixed bottom-0 end-0 translate-middle d-none&#039; (tried for 3 second(s) with 500 milliseconds interval)
Build info: version: &#039;4.15.0&#039;, revision: &#039;1d14b5521b&#039;
System info: os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;19.0.2&#039;
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 123.0.6312.86, chrome: {chromedriverVersion: 122.0.6261.94 (880dbf29479c..., userDataDir: C:\Users\JOBIAK\AppData\Loc...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:58252}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: Windows 10, proxy: Proxy(), se:bidiEnabled: false, se:cdp: ws://192.168.0.7:4444/sessi..., se:cdpVersion: 123.0.6312.86, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: fe26225836db26e3cb17861eea70bda9" type="org.openqa.selenium.TimeoutException">
      <![CDATA[org.openqa.selenium.TimeoutException: Expected condition failed: waiting for Attribute or property 'class' to be 'd-none'. Current value: 'btn theme-btn py-2 px-3 position-fixed bottom-0 end-0 translate-middle d-none' (tried for 3 second(s) with 500 milliseconds interval)
Build info: version: '4.15.0', revision: '1d14b5521b'
System info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '19.0.2'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 123.0.6312.86, chrome: {chromedriverVersion: 122.0.6261.94 (880dbf29479c..., userDataDir: C:\Users\JOBIAK\AppData\Loc...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:58252}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: Windows 10, proxy: Proxy(), se:bidiEnabled: false, se:cdp: ws://192.168.0.7:4444/sessi..., se:cdpVersion: 123.0.6312.86, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: fe26225836db26e3cb17861eea70bda9
at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:84)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:229)
at pageElements.AbstractComponents.waitUsingAttribute(AbstractComponents.java:50)
at scripts.HomePage_T_C.verifyScrollButton(HomePage_T_C.java:60)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1589)
]]>
    </error>
  </testcase> <!-- verifyScrollButton -->
  <testcase classname="scripts.HomePage_T_C" name="verifyRobotsBeforeSubmission" time="8.269">
    <failure message="The following asserts failed:
	expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at scripts.HomePage_T_C.verifyRobotsBeforeSubmission(HomePage_T_C.java:20)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1589)
]]>
    </failure>
  </testcase> <!-- verifyRobotsBeforeSubmission -->
  <testcase classname="scripts.HomePage_T_C" name="verifyOgSiteName" time="25.234">
    <failure message="The following asserts failed:
	Domain name:Hiring Event og:sitename: hiring-event expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Domain name:Hiring Event og:sitename: hiring-event expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at scripts.HomePage_T_C.verifyOgSiteName(HomePage_T_C.java:139)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1589)
]]>
    </failure>
  </testcase> <!-- verifyOgSiteName -->
  <testcase classname="scripts.HomePage_T_C" name="verifySitemap" time="15.756"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyImageFormat" time="16.507">
    <failure message="The following asserts failed:
	https://jobsnotion.com/design-folder/pictures/company-logos/jn-live-trucking-logo.png expected [true] but found [false],
	https://jobsnotion.com/design-folder/pictures/icons/location.svg expected [true] but found [false],
	https://jobsnotion.com/design-folder/pictures/icons/clock.svg expected [true] but found [false],
	https://jobsnotion.com/design-folder/pictures/company-logos/jn-petsmart-logo.png expected [true] but found [false],
	https://jobsnotion.com/design-folder/pictures/icons/location.svg expected [true] but found [false],
	https://jobsnotion.com/design-folder/pictures/company-logos/jn-federal-bureau-of-investigation-logo.png expected [true] but found [false],
	https://jobsnotion.com/design-folder/pictures/icons/location.svg expected [true] but found [false],
	https://jobsnotion.com/design-folder/pictures/company-logos/jn-superprof-logo.png expected [true] but found [false],
	https://jobsnotion.com/design-folder/pictures/icons/location.svg expected [true] but found [false],
	https://jobsnotion.com/design-folder/pictures/icons/clock.svg expected [true] but found [false],
	https://jobsnotion.com/design-folder/pictures/company-logos/jn-autozone-logo.png expected [true] but found [false],
	https://jobsnotion.com/design-folder/pictures/icons/location.svg expected [true] but found [false],
	https://jobsnotion.com/design-folder/pictures/icons/clock.svg expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
https://jobsnotion.com/design-folder/pictures/company-logos/jn-live-trucking-logo.png expected [true] but found [false],
https://jobsnotion.com/design-folder/pictures/icons/location.svg expected [true] but found [false],
https://jobsnotion.com/design-folder/pictures/icons/clock.svg expected [true] but found [false],
https://jobsnotion.com/design-folder/pictures/company-logos/jn-petsmart-logo.png expected [true] but found [false],
https://jobsnotion.com/design-folder/pictures/icons/location.svg expected [true] but found [false],
https://jobsnotion.com/design-folder/pictures/company-logos/jn-federal-bureau-of-investigation-logo.png expected [true] but found [false],
https://jobsnotion.com/design-folder/pictures/icons/location.svg expected [true] but found [false],
https://jobsnotion.com/design-folder/pictures/company-logos/jn-superprof-logo.png expected [true] but found [false],
https://jobsnotion.com/design-folder/pictures/icons/location.svg expected [true] but found [false],
https://jobsnotion.com/design-folder/pictures/icons/clock.svg expected [true] but found [false],
https://jobsnotion.com/design-folder/pictures/company-logos/jn-autozone-logo.png expected [true] but found [false],
https://jobsnotion.com/design-folder/pictures/icons/location.svg expected [true] but found [false],
https://jobsnotion.com/design-folder/pictures/icons/clock.svg expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at scripts.HomePage_T_C.verifyImageFormat(HomePage_T_C.java:182)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1589)
]]>
    </failure>
  </testcase> <!-- verifyImageFormat -->
  <testcase classname="scripts.HomePage_T_C" name="verifyFolderNameInPath" time="16.102"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyOgImageRepetition" time="22.803"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyProtocol" time="7.131"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyScrollButton" time="12.235">
    <error message="Expected condition failed: waiting for Attribute or property &#039;class&#039; to be &#039;d-none&#039;. Current value: &#039;scroll-top btn-hover border-0 py-2 px-3 position-fixed bottom-0 end-0 translate-middle&#039; (tried for 3 second(s) with 500 milliseconds interval)
Build info: version: &#039;4.15.0&#039;, revision: &#039;1d14b5521b&#039;
System info: os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;19.0.2&#039;
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 123.0.6312.86, chrome: {chromedriverVersion: 122.0.6261.94 (880dbf29479c..., userDataDir: C:\Users\JOBIAK\AppData\Loc...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:58022}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: Windows 10, proxy: Proxy(), se:bidiEnabled: false, se:cdp: ws://192.168.0.7:4444/sessi..., se:cdpVersion: 123.0.6312.86, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: acff26d85985c650e46d56861dfea56a" type="org.openqa.selenium.TimeoutException">
      <![CDATA[org.openqa.selenium.TimeoutException: Expected condition failed: waiting for Attribute or property 'class' to be 'd-none'. Current value: 'scroll-top btn-hover border-0 py-2 px-3 position-fixed bottom-0 end-0 translate-middle' (tried for 3 second(s) with 500 milliseconds interval)
Build info: version: '4.15.0', revision: '1d14b5521b'
System info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '19.0.2'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 123.0.6312.86, chrome: {chromedriverVersion: 122.0.6261.94 (880dbf29479c..., userDataDir: C:\Users\JOBIAK\AppData\Loc...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:58022}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: Windows 10, proxy: Proxy(), se:bidiEnabled: false, se:cdp: ws://192.168.0.7:4444/sessi..., se:cdpVersion: 123.0.6312.86, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: acff26d85985c650e46d56861dfea56a
at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:84)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:229)
at pageElements.AbstractComponents.waitUsingAttribute(AbstractComponents.java:50)
at scripts.HomePage_T_C.verifyScrollButton(HomePage_T_C.java:60)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1589)
]]>
    </error>
  </testcase> <!-- verifyScrollButton -->
  <testcase classname="scripts.HomePage_T_C" name="verifySitemap" time="6.394"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyMetaContent" time="30.416">
    <error message="Cannot invoke &quot;java.util.List.get(int)&quot; because the return value of &quot;java.util.Map.get(Object)&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "java.util.List.get(int)" because the return value of "java.util.Map.get(Object)" is null
at utilities.MetaFileHandler.readMetaContentFromExcel(MetaFileHandler.java:99)
at scripts.HomePage_T_C.verifyMetaContent(HomePage_T_C.java:88)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1589)
]]>
    </error>
  </testcase> <!-- verifyMetaContent -->
  <testcase classname="scripts.HomePage_T_C" name="verifyTitleRepetition" time="7.440"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyNetworkStatus" time="9.936"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyNetworkStatus" time="10.151"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyDomainViewPort" time="9.407"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyOgImageNetwork" time="14.835"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyFolderNameInPath" time="21.749"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyTitleRepetition" time="10.974"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyOgImageRepetition" time="9.513"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyRobotsBeforeSubmission" time="14.053">
    <failure message="The following asserts failed:
	expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at scripts.HomePage_T_C.verifyRobotsBeforeSubmission(HomePage_T_C.java:20)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1589)
]]>
    </failure>
  </testcase> <!-- verifyRobotsBeforeSubmission -->
  <testcase classname="scripts.HomePage_T_C" name="verifySubdomain" time="16.761"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyHttpInPageSource" time="15.458"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyFolderNameInPath" time="26.013"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyImageFormat" time="10.908">
    <failure message="The following asserts failed:
	https://openpostions.io/op-theme/images/icons/tick.svg expected [true] but found [false],
	https://openpostions.io/op-theme/images/icons/tick.svg expected [true] but found [false],
	https://openpostions.io/op-theme/images/icons/tick.svg expected [true] but found [false],
	https://openpostions.io/op-theme/images/company-logos/rnjz-live-trucking-logo.png expected [true] but found [false],
	https://openpostions.io/op-theme/images/company-logos/rnjz-live-trucking-logo.png expected [true] but found [false],
	https://openpostions.io/op-theme/images/company-logos/rnjz-live-trucking-logo.png expected [true] but found [false],
	https://openpostions.io/op-theme/images/company-logos/rnjz-live-trucking-logo.png expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
https://openpostions.io/op-theme/images/icons/tick.svg expected [true] but found [false],
https://openpostions.io/op-theme/images/icons/tick.svg expected [true] but found [false],
https://openpostions.io/op-theme/images/icons/tick.svg expected [true] but found [false],
https://openpostions.io/op-theme/images/company-logos/rnjz-live-trucking-logo.png expected [true] but found [false],
https://openpostions.io/op-theme/images/company-logos/rnjz-live-trucking-logo.png expected [true] but found [false],
https://openpostions.io/op-theme/images/company-logos/rnjz-live-trucking-logo.png expected [true] but found [false],
https://openpostions.io/op-theme/images/company-logos/rnjz-live-trucking-logo.png expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at scripts.HomePage_T_C.verifyImageFormat(HomePage_T_C.java:182)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1589)
]]>
    </failure>
  </testcase> <!-- verifyImageFormat -->
  <testcase classname="scripts.HomePage_T_C" name="verifyOgSiteName" time="18.283">
    <failure message="The following asserts failed:
	Domain name:Open Postions og:sitename: openpostions expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Domain name:Open Postions og:sitename: openpostions expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at scripts.HomePage_T_C.verifyOgSiteName(HomePage_T_C.java:139)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1589)
]]>
    </failure>
  </testcase> <!-- verifyOgSiteName -->
  <testcase classname="scripts.HomePage_T_C" name="verifyDomainViewPort" time="17.161"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyProtocol" time="12.417"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyStyleSheets" time="16.999">
    <failure message="The following asserts failed:
	expected [false] but found [true]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [false] but found [true]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at scripts.HomePage_T_C.verifyStyleSheets(HomePage_T_C.java:232)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1589)
]]>
    </failure>
  </testcase> <!-- verifyStyleSheets -->
  <testcase classname="scripts.HomePage_T_C" name="verifyFaviconNetwork" time="22.035"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyOgImageRepetition" time="8.584"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyOgSiteName" time="10.999">
    <failure message="The following asserts failed:
	Domain name:Jobs Notion og:sitename: jobsnotion expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Domain name:Jobs Notion og:sitename: jobsnotion expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at scripts.HomePage_T_C.verifyOgSiteName(HomePage_T_C.java:139)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1589)
]]>
    </failure>
  </testcase> <!-- verifyOgSiteName -->
  <testcase classname="scripts.HomePage_T_C" name="verifyStyleSheets" time="21.484"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyDomainPipeline" time="18.934"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyConsole" time="12.259"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyConsole" time="6.663"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyImageFormat" time="15.687">
    <failure message="The following asserts failed:
	https://hiring-event.com/asset-folder/images/icons/dynamic-platform.svg expected [true] but found [false],
	https://hiring-event.com/asset-folder/images/icons/expert.svg expected [true] but found [false],
	https://hiring-event.com/asset-folder/images/icons/tech.svg expected [true] but found [false],
	https://hiring-event.com/asset-folder/images/icons/advantage.svg expected [true] but found [false],
	https://hiring-event.com/asset-folder/images/icons/vision.svg expected [true] but found [false],
	https://hiring-event.com/asset-folder/images/icons/journey.svg expected [true] but found [false],
	https://hiring-event.com/asset-folder/images/company-logos/usps-logo.png expected [true] but found [false],
	https://hiring-event.com/asset-folder/images/icons/location.svg expected [true] but found [false],
	https://hiring-event.com/asset-folder/images/company-logos/sams-club-logo.png expected [true] but found [false],
	https://hiring-event.com/asset-folder/images/icons/location.svg expected [true] but found [false],
	https://hiring-event.com/asset-folder/images/company-logos/hogan-transportation-logo.jpg expected [true] but found [false],
	https://hiring-event.com/asset-folder/images/icons/location.svg expected [true] but found [false],
	https://hiring-event.com/asset-folder/images/company-logos/usps-logo.png expected [true] but found [false],
	https://hiring-event.com/asset-folder/images/icons/location.svg expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
https://hiring-event.com/asset-folder/images/icons/dynamic-platform.svg expected [true] but found [false],
https://hiring-event.com/asset-folder/images/icons/expert.svg expected [true] but found [false],
https://hiring-event.com/asset-folder/images/icons/tech.svg expected [true] but found [false],
https://hiring-event.com/asset-folder/images/icons/advantage.svg expected [true] but found [false],
https://hiring-event.com/asset-folder/images/icons/vision.svg expected [true] but found [false],
https://hiring-event.com/asset-folder/images/icons/journey.svg expected [true] but found [false],
https://hiring-event.com/asset-folder/images/company-logos/usps-logo.png expected [true] but found [false],
https://hiring-event.com/asset-folder/images/icons/location.svg expected [true] but found [false],
https://hiring-event.com/asset-folder/images/company-logos/sams-club-logo.png expected [true] but found [false],
https://hiring-event.com/asset-folder/images/icons/location.svg expected [true] but found [false],
https://hiring-event.com/asset-folder/images/company-logos/hogan-transportation-logo.jpg expected [true] but found [false],
https://hiring-event.com/asset-folder/images/icons/location.svg expected [true] but found [false],
https://hiring-event.com/asset-folder/images/company-logos/usps-logo.png expected [true] but found [false],
https://hiring-event.com/asset-folder/images/icons/location.svg expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at scripts.HomePage_T_C.verifyImageFormat(HomePage_T_C.java:182)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1589)
]]>
    </failure>
  </testcase> <!-- verifyImageFormat -->
  <testcase classname="scripts.HomePage_T_C" name="verifyProtocol" time="10.684"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyDescRepetition" time="8.805"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyHttpInPageSource" time="14.462"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyFaviconNetwork" time="8.113"/>
  <testcase classname="scripts.HomePage_T_C" name="verifySitemap" time="5.076"/>
  <testcase classname="scripts.HomePage_T_C" name="verifySubdomain" time="28.299"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyScrollButton" time="14.591">
    <error message="Expected condition failed: waiting for Attribute or property &#039;class&#039; to be &#039;d-none&#039;. Current value: &#039;btn theme-btn py-2 px-3 rounded-20 position-fixed bottom-0 end-0 translate-middle d-none&#039; (tried for 3 second(s) with 500 milliseconds interval)
Build info: version: &#039;4.15.0&#039;, revision: &#039;1d14b5521b&#039;
System info: os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;19.0.2&#039;
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 123.0.6312.86, chrome: {chromedriverVersion: 122.0.6261.94 (880dbf29479c..., userDataDir: C:\Users\JOBIAK\AppData\Loc...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:58115}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: Windows 10, proxy: Proxy(), se:bidiEnabled: false, se:cdp: ws://192.168.0.7:4444/sessi..., se:cdpVersion: 123.0.6312.86, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: be81b0d496a067228140c2575a56555a" type="org.openqa.selenium.TimeoutException">
      <![CDATA[org.openqa.selenium.TimeoutException: Expected condition failed: waiting for Attribute or property 'class' to be 'd-none'. Current value: 'btn theme-btn py-2 px-3 rounded-20 position-fixed bottom-0 end-0 translate-middle d-none' (tried for 3 second(s) with 500 milliseconds interval)
Build info: version: '4.15.0', revision: '1d14b5521b'
System info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '19.0.2'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 123.0.6312.86, chrome: {chromedriverVersion: 122.0.6261.94 (880dbf29479c..., userDataDir: C:\Users\JOBIAK\AppData\Loc...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:58115}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: Windows 10, proxy: Proxy(), se:bidiEnabled: false, se:cdp: ws://192.168.0.7:4444/sessi..., se:cdpVersion: 123.0.6312.86, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: be81b0d496a067228140c2575a56555a
at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:84)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:229)
at pageElements.AbstractComponents.waitUsingAttribute(AbstractComponents.java:50)
at scripts.HomePage_T_C.verifyScrollButton(HomePage_T_C.java:60)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1589)
]]>
    </error>
  </testcase> <!-- verifyScrollButton -->
  <testcase classname="scripts.HomePage_T_C" name="verifyOgUrlRepetition" time="22.302"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyDescRepetition" time="9.072"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyScrollButton" time="19.529">
    <error message="Expected condition failed: waiting for Attribute or property &#039;class&#039; to be &#039;d-none&#039;. Current value: &#039;btn theme-btn py-2 px-3 position-fixed bottom-0 end-0 translate-middle d-none&#039; (tried for 3 second(s) with 500 milliseconds interval)
Build info: version: &#039;4.15.0&#039;, revision: &#039;1d14b5521b&#039;
System info: os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;19.0.2&#039;
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 123.0.6312.86, chrome: {chromedriverVersion: 122.0.6261.94 (880dbf29479c..., userDataDir: C:\Users\JOBIAK\AppData\Loc...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:58205}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: Windows 10, proxy: Proxy(), se:bidiEnabled: false, se:cdp: ws://192.168.0.7:4444/sessi..., se:cdpVersion: 123.0.6312.86, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: b826d198d36b5a0c63a0533015d27a56" type="org.openqa.selenium.TimeoutException">
      <![CDATA[org.openqa.selenium.TimeoutException: Expected condition failed: waiting for Attribute or property 'class' to be 'd-none'. Current value: 'btn theme-btn py-2 px-3 position-fixed bottom-0 end-0 translate-middle d-none' (tried for 3 second(s) with 500 milliseconds interval)
Build info: version: '4.15.0', revision: '1d14b5521b'
System info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '19.0.2'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 123.0.6312.86, chrome: {chromedriverVersion: 122.0.6261.94 (880dbf29479c..., userDataDir: C:\Users\JOBIAK\AppData\Loc...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:58205}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: Windows 10, proxy: Proxy(), se:bidiEnabled: false, se:cdp: ws://192.168.0.7:4444/sessi..., se:cdpVersion: 123.0.6312.86, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: b826d198d36b5a0c63a0533015d27a56
at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:84)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:229)
at pageElements.AbstractComponents.waitUsingAttribute(AbstractComponents.java:50)
at scripts.HomePage_T_C.verifyScrollButton(HomePage_T_C.java:60)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1589)
]]>
    </error>
  </testcase> <!-- verifyScrollButton -->
  <testcase classname="scripts.HomePage_T_C" name="verifyOgSiteName" time="14.656"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyConsole" time="15.536"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyFolderNameInPath" time="15.588"/>
  <testcase classname="scripts.HomePage_T_C" name="verifyDomainViewPort" time="16.417"/>
</testsuite> <!-- scripts.HomePage_T_C -->
